rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -15,6 +15,10 @@ bin/elasticsearch -Enode.attr.testattr=test -Epath.repo=/tmp -Erepositories.url.
 
 =======================================
 
+Test execution
+---------------
+Please refer to https://github.com/elastic/elasticsearch/blob/master/TESTING.asciidoc#testing-the-rest-layer[this section].
```

Comment:
Let's use relative links so we aren't hard-coding any particular branch.

```suggestion
Please refer to /TESTING.asciidoc#testing-the-rest-layer[this section].
```

Reply:
@mark-vieira Thanks, fixed :)